model/map/src/main/java/org/keycloak/models/map/storage/tree/EmptyMapStorage.java

```diff
@@ -27,18 +25,10 @@
  *
  * @author hmlnarik
  */
-public class EmptyMapStorage<V extends AbstractEntity, M> implements MapStorage<V, M> {
+public class EmptyMapStorage {
```

Comment:
This is intentional. Previously the `getInstance` returned `MapStorage`. Each call to `createTransaction` from the `INSTANCE` created a new transaction instance. 

After this PR `getInstance` returns the transaction (in old naming), and ' MapStorage` does not exist. I know it is confusing because the transaction was renamed to `MapStorage` but the number of objects created should be the same